.eslintrc.js

```diff
@@ -32,14 +32,6 @@ module.exports = {
     '@typescript-eslint/no-inferrable-types': [
       'error',
       { ignoreProperties: true }
-    ],
-
-    'sort-imports': [
-      'warn',
-      {
-        ignoreMemberSort: true,
-        memberSyntaxSortOrder: ['none', 'all', 'single', 'multiple']
-      }
```

Comment:
I would like to keep that eslint warning for unsorted imports. The fix is just to run `npm run format`.

Reply:
Really? But how do you want to auto-format then natively? What's wrong with the native API sorting?

Reply:
`npm run format` doesn't solve this warnings
The configuration doesn't satisfy TS native API ordering

Reply:
The eslint setup for that rule is the same of the sorting ordering expected with the native API. Eslint does not sort any imports with it, it's just warn that it should be done.

Reply:
Ok, but then something is configured incorrect

<img src="https://user-images.githubusercontent.com/7195563/107400105-cf2e7600-6b01-11eb-9470-2800b3b5f964.png" width="400px"/>

Reply:
Oh ... I see you cannot see these warnings in master because they are in #92 

<img src="https://user-images.githubusercontent.com/7195563/107402256-0c940300-6b04-11eb-8516-d693763d538a.png" width="400px"/>

But all these 9 warnings are false positives and conflicts with the native ordering by TS native
I think the problem is, that this rule is only for javascript but not for typescript :confused:  
Maybe we should move it to the js override block?

Reply:
That setup is working:

```
    'sort-imports': [
      'warn',
      {
        ignoreMemberSort: true,
        ignoreDeclarationSort: true
      }
    ]
```

Could you please fix it in that PR?